package.json

```diff
@@ -51,29 +51,29 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^6.1.8",
+    "@angular/animations": "^7.0.1",
     "@angular/cdk": "^6.4.7",
-    "@angular/common": "^6.1.8",
-    "@angular/compiler": "^6.1.8",
-    "@angular/core": "^6.1.8",
-    "@angular/forms": "^6.1.8",
-    "@angular/http": "^6.1.8",
+    "@angular/common": "^7.0.1",
+    "@angular/compiler": "^7.0.1",
+    "@angular/core": "^7.0.1",
+    "@angular/forms": "^7.0.1",
+    "@angular/http": "^7.0.1",
     "@angular/material": "^6.4.7",
-    "@angular/platform-browser": "^6.1.8",
-    "@angular/platform-browser-dynamic": "^6.1.8",
-    "@angular/platform-server": "^6.1.8",
-    "@angular/router": "^6.1.8",
+    "@angular/platform-browser": "^7.0.1",
+    "@angular/platform-browser-dynamic": "^7.0.1",
+    "@angular/platform-server": "^7.0.1",
+    "@angular/router": "^7.0.1",
     "core-js": "^2.5.7",
-    "rxjs": "^6.3.2",
+    "rxjs": "^6.3.3",
     "web-animations-js": "^2.3.1",
     "zone.js": "^0.8.26"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "^0.8.3",
+    "@angular-devkit/build-angular": "~0.10.0",
     "@angular-devkit/build-ng-packagr": "^0.8.3",
```

Comment:
What about this package? There's `0.10.3` available on npm. Maybe we should also update this one.

Reply:
@d3lm @markgoho I am really sorry. I did not saw the ongoing conversation on twitter between both of you.

@d3lm Pick whatever PR you like. It did not take me long to do this.
Nevertheless, I will update the dependency you have mentioned.

Sorry for confusing everyone. 🙏

Reply:
Updated ng-packagr